### PR TITLE
Update samples to use the latest versions of the python SDK

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1607)

### CODE REVIEW

File: samples/ai-platform/Chicago Crime Pipeline.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "%%capture\n",
     "\n",
     "# Install the SDK (Uncomment the code if the SDK is not installed before)\n",
-    "KFP_PACKAGE = 'https://storage.googleapis.com/ml-pipeline/release/0.1.22/kfp.tar.gz'\n",
+    "KFP_PACKAGE = 'kfp'\n",
```
Comment (neuromage) [inline samples/ai-platform/Chicago Crime Pipeline.ipynb]: How about we just remove this variable altogether, and have the line read everywhere as:
`!pip3 install kfp --upgrade`

Comment (neuromage) [reply samples/ai-platform/Chicago Crime Pipeline.ipynb]: It'll also be more consistent across notebooks.

Comment (animeshsingh) [reply samples/ai-platform/Chicago Crime Pipeline.ipynb]: +1. Its becoming cumbersome maintaining like this

Comment (gaoning777) [reply samples/ai-platform/Chicago Crime Pipeline.ipynb]: SG.

**Comment by Ark-kun** [comment]
/lgtm
We need to make sure that the package on PyPI is always up to date.

**Comment by gaoning777** [comment]
Pypi release is already one step in the release playbook.

**Pull request merged**